### PR TITLE
fixed quotation marks, added title

### LINKED ISSUE #48: Carousel Quote title should be part of the block
title

### PR DESCRIPTION
**Pull request by badvision**
Fix #48 

Test URLs:
- Before: https://main--hsf-commonmoves--hlxsites.hlx.page/drafts/carousel
- After: https://quote-title--hsf-commonmoves--hlxsites.hlx.page/drafts/carousel

### CODE REVIEW

File: blocks/quote-carousel/quote-carousel.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ async function getContent(block) {
 export default async function decorate(block) {
   const blockId = crypto.randomUUID();
   const content = await getContent(block);
+  const title = block.querySelector('div > div > div > div > div').textContent;
```
Comment (bstopp) [inline blocks/quote-carousel/quote-carousel.js:112]: This should be based on row, not column.

First row is title, second row is the link to the sheet with the quotes.

**Review by bstopp — CHANGES_REQUESTED** [review]

File: blocks/quote-carousel/quote-carousel.js [hunk 2]
```diff
@@ -106,21 +18,58 @@ async function getContent(block) {
   return data;
 }
 
+function getTitle(block) {
+  const titleElem = [...block.querySelectorAll('div')]
+    .filter((e) => e.innerText.toLowerCase() === 'title');
+  return titleElem.length > 0 ? titleElem[0].nextElementSibling.innerText : '';
+}
+
+let alreadyDeferred = false;
```
Comment (bstopp) [inline blocks/quote-carousel/quote-carousel.js:27]: I assume this is to check if the delayed JS is already loaded?

**Review by bstopp — CHANGES_REQUESTED** [review]
The title doesn't rotate with the carousel itself. It's also not included inside the box when in mobile view.

**Comment by bstopp** [comment]
Also, things are off center on some viewports
Seller_Testimonials

**Comment by badvision** [comment]
> The title doesn't rotate with the carousel itself. It's also not included inside the box when in mobile view.

Crud!  Ok that is good to know.  I'll work on that.

**Review by bstopp — APPROVED** [review]

**Review by bstopp — APPROVED** [review]

**Pull request merged**